.azure-pipelines/signing.yml

```diff
@@ -7,6 +7,14 @@ parameters:
 
 steps:
 - ${{ if parameters.isWindows }}:
+  - task: PowerShell@2
+    inputs:
+      targetType: 'filePath'
+      filePath: ./.azure-pipelines/scripts/Unsignig-ThirdPartyAssemlies.ps1
+      arguments: '-LayoutRoot ${{ parameters.layoutRoot }}'
+      ignoreLASTEXITCODE: true
```

Comment:
Why do we need to ignore LASTEXITCODE?

Reply:
It is a rudiment from the previous solution, removed this option for now